### DISCUSSION TITLE
I'd like to purchase a Plus version, but my phone is LineageOS de-googled

### DISCUSSION THREAD

**Discussion by martan1912** [osmandapp/OsmAnd#18858] [Q&A]
Hello, I'd like to use this app on my LineageOS Android phone that has no google services  (including Google Play store) installed. I did install free version and it works , but I'd like to get TOPO maps and other features which are paid.  How do I make the purchase if I don't have Google play ?

**Comment by DmitryAlexei** [comment]
Purchase can also be made in Huawei and Amazon stores
